### PR TITLE
made component for font color

### PR DESCRIPTION
**Pull request by dnori817**

### CODE REVIEW

File: js/App.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+const FontColor = require("./components/FontColor");
+
 
 class App extends React.Component {
 	render() {
 		return (
 			<div className="app">
 				<Title>Test Title</Title>
 				<Description library="will.js"/>
+				<FontColor color="red"/>
```
Comment (wbobeirne) [inline js/App.js]: FontColor should take in children, like Title.

File: js/components/FontColor.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const React = require("react");
+
+class FontColor extends React.Component {
+	render() {
+		const colorName = this.props.color;
+		return (
+			<font className="font" color={colorName}>I am a {colorName} font</font>
```
Comment (wbobeirne) [inline js/components/FontColor.js]: Nice work on the property! Just have it render its children ala title as well.

**Review by wbobeirne — CHANGES_REQUESTED** [review]

**Comment by dnori817** [comment]
Like this?

**Review by wbobeirne — APPROVED** [review]
Precisely, nice work!

**Pull request closed without merging**